src/System.CommandLine/Invocation/CliCommandAction.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.CommandLine.Invocation;
+
+// FIX: separate files
+
+public abstract class CliAction
```

Comment:
Do we want the filename to match the class

Reply:
They will once split out into separate files.